api/npm.ts

```diff
@@ -69,17 +69,15 @@ async function handler ({ topic, scope, pkg, tag }: PathArgs) {
 
 async function npmMetadata (pkg: string, ver = 'latest'): Promise<any> {
   const host = process.env.NPM_REGISTRY || "https://registry.npmjs.org"
-  if (pkg[0] === "@") {
+  if (pkg[0] === "@" || ver !== 'latest') {
```

Comment:
what about `.charAt(0)`? @amio thoughts?

Reply:
Guess I might choose `.startsWith()`, but all fine to me :D